demo-kotlin/app/src/main/java/com/example/placesdemo/programmatic_predictions/ProgrammaticAutocompleteToolbarActivity.kt

```diff
@@ -131,7 +176,51 @@ class ProgrammaticAutocompleteToolbarActivity : AppCompatActivity() {
         }
     }
 
+
+    /**
+     * Performs a Geocode API request and displays the result in a dialog
```

Comment:
Geocode API -> Geocoding API
Perhaps add a helpful link to https://developers.google.com/maps/documentation/geocoding/intro